packages/vue/src/components/organisms/SfProductCard/SfProductCard.vue

```diff
@@ -27,6 +27,24 @@
           />
         </SfLink>
       </slot>
+      <slot name="colors" v-bind="{ colors }">
+        <SfColorPicker
+          v-if="colors.length"
+          class="sf-product-card__colors"
+          label="Choose color"
+          :is-open="!isMobile || openColorPicker"
+          @click:toggle="toggleColorPicker"
+        >
+          <SfColor
+            v-for="(color, i) in colors"
+            :key="color.value"
+            :color="color.color"
+            :selected="color.selected"
+            class="sf-product-card__color"
+            @click="handleSelectedColor(i)"
```

Comment:
Can't we just emit the event with color here?
```suggestion
            @click="$emit("click:colors", color)"
```